### PR TITLE
Converted the entire Java code to Kotlin.

### PR DESCRIPTION
**Pull request by Srishtikumari9**
Java code converted to Kotlin.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -39,4 +39,41 @@
         </activity>
     </application>
 
-</manifest>
\ No newline at end of file
+</manifest>
```
Comment (pawankgupta-se) [inline app/src/main/AndroidManifest.xml:41]: Remove these extra lines.

File: app/src/main/java/com/example/socialapp/models/Post.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.example.socialapp.models
+import com.google.gson.annotations.SerializedName
+class Post {
```
Comment (pawankgupta-se) [inline app/src/main/java/com/example/socialapp/models/Post.kt]: Should use data class.

**Review by pawankgupta-se — APPROVED** [review]

File: app/src/main/java/com/example/socialapp/models/Post.kt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    @SerializedName("name")
+    var name: String? = null
+
+    @SerializedName("id")
+    var id: String? = null
+    override fun toString(): String {
```
Comment (pawankgupta-se) [inline app/src/main/java/com/example/socialapp/models/Post.kt]: data class will not have toString() method

**Review by pawankgupta-se — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Srishtikumari9) [reply app/src/main/AndroidManifest.xml:41]: Changes done.

File: app/src/main/java/com/example/socialapp/models/Post.kt (hunk 2 above)
Comment (Srishtikumari9) [reply app/src/main/java/com/example/socialapp/models/Post.kt]: Changes done.

File: app/src/main/java/com/example/socialapp/models/Post.kt (hunk 3 above)
Comment (Srishtikumari9) [reply app/src/main/java/com/example/socialapp/models/Post.kt]: Changes done.

**Review by pawankgupta-se — APPROVED** [review]

**Pull request merged**